ext/IntegralsForwardDiffExt.jl

```diff
@@ -29,7 +29,8 @@ function Integrals.__solvebp(cache, alg, sensealg, lb, ub,
         dfdp = function (out, x, p)
             dualp = reinterpret(ForwardDiff.Dual{T, V, P}, p)
             if cache.batch > 0
-                dx = similar(dualp, cache.nout, size(x, 2))
+                dx = cache.nout == 1 ? similar(dualp, size(x, ndims(x))) :
```

Comment:
won't the nout = 1 case not be able to use `similar` because it could be scalar?